### ISSUE TITLE
Create more friendly build targets (Trac #30)

### ISSUE THREAD

**Issue by jamesog** [jamesog/boxbackup#30]
Currently the build scripts generate the Makefile which makes build targets called

 * parcels/''$product_name''-''$product_version''-client-''$target_os''
 * parcels/''$product_name''-''$product_version''-server-''$target_os''

These aren't very nice to type when building just one component. The attached patch makes two new targets (build-backup-client, build-backup-server) which depend on these existing ones.

Migrated from https://www.boxbackup.org/ticket/30
```json
{
    "status": "closed", 
    "changetime": "2007-06-30T13:32:54", 
    "description": "Currently the build scripts generate the Makefile which makes build targets called\n\n * parcels/''$product_name''-''$product_version''-client-''$target_os''\n * parcels/''$product_name''-''$product_version''-server-''$target_os''\n\nThese aren't very nice to type when building just one component. The attached patch makes two new targets (build-backup-client, build-backup-server) which depend on these existing ones.", 
    "reporter": "jamesog", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1183210374000000", 
    "component": "scripts", 
    "summary": "Create more friendly build targets", 
    "priority": "normal", 
    "keywords": "Makefile, scripts", 
    "version": "trunk", 
    "time": "2007-06-29T11:57:48", 
    "milestone": "0.11", 
    "owner": "", 
    "type": "enhancement"
}
```

**Comment by jamesog** [comment]
Trac update at 20070629T11:58:34: 
* jamesog changed attachment from "" to "patch-infrastructure-makeparcels.pl.in"
* jamesog commented:

> makeparcels.pl.in patch

**Comment by jamesog** [comment]
Trac update at 20070630T11:22:08: 
* chris commented:

> (In [1709]) Improved build targets, thanks to James O'Gorman (fixes #30)


* chris changed status from "new" to "closed"
* chris changed resolution from "" to "fixed"

**Comment by jamesog** [comment]
Trac update at 20070630T11:22:57: chris commented:

> (In [1710]) Improved build targets, thanks to James O'Gorman (merges [1709], fixes #30)

**Comment by jamesog** [comment]
Trac update at 20070630T13:32:54: jamesog commented:

> (In [1712]) Implement two new build targets:
>  * build-backup-client
>  * build-backup-server
> 
> These depend on the existing client/server parcel targets but are easier on the eye.
> Refs and fixes #30.
